cgogn/modeling/algos/subdivision_utils.h

```diff
@@ -153,8 +153,8 @@ inline IncidenceGraph::Vertex quadrangulate_face(IncidenceGraph& ig, IncidenceGr
 		edges.push_back(Edge(new_edges[ev1.index_]));
 		edges.push_back(Edge(new_edges[ev0.index_]));
 		// create the face
-		Face f = add_face(ig, edges);
-		sort_face_edges(ig, f);
+		Face nf = add_face(ig, edges);
```

Comment:
There's already a function parameter `f` so I figured I might as well unshadow this; `nf` stands for "new face"